loading/lazyload/META.yml

```diff
@@ -0,0 +1,3 @@
+spec: https://github.com/whatwg/html/pull/3752
```

Comment:
Ah, in that case this should be in the html/ directory and be a tentative test.

Reply:
Do you mean META.yml and all the tests in loading/lazyload should move to html/?
Also it feels like the META.yml should be a separate issue.

Reply:
No META.yml is needed if the tests are put in the appropriate subdirectoriy of html/. What subdirectory depends on what part of the spec this will be in, I didn't check that.

Reply:
Makes sense that we'd move all of the tests to a (probably new) directory in html/. Could we do that after landing these tests, in a separate PR?

Reply:
Sure.